analytics-webapps/src/main/webapp/vue-app/generic-portlet/components/chart/AnalyticsChart.vue

```diff
@@ -110,6 +110,14 @@ export default {
           const serie = {
```

Comment:
it's better to add this sort operation before the forEach :
`        charts.sort((chartData1, chartData2) => Math.max(...chartData2.values) - Math.max(...chartData1.values));`
this way we ensure at least that the biggest chart doesn't hide the chart with lower values